integration-tests/README.md

```diff
@@ -123,6 +123,27 @@ Druid routers for security group integration test (permissive tls, no client aut
    mvn verify -P integration-tests -pl integration-tests -Ddocker.build.skip=true -Ddocker.run.skip=true -Dit.test=<test_name>
    ```
 
+### Running tests from IntelliJ
+
+A prerequisite to run tests from intellij you need a cluster up and running. Modify the run configuration's `VM options`
+to the values below.
```

Comment:
```suggestion
Before running tests from IntelliJ, ensure you have a Druid cluster running.
In IntelliJ, edit configurations to add the following VM options:
```

Reply:
Reads a lot better. Will just make slight naming changes to make sure it matches exactly with intellij's naming